terraform/modules/enclave/prometheus/test/integration/prometheus/controls/operanting_systems.rb

```diff
@@ -0,0 +1,60 @@
+control "operating_system" do
```

Comment:
Just spotted that this file is misspelt as `operanting_systems.rb` as opposed to `operating_systems.rb`